recipes/triton/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "1.1.2" %}
+
+package:
+  name: triton
+  version: {{ version }}
+
+source:
+  url: https://github.com/openai/triton/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 44040696460f1d02a7251dd59468d7405982b55e112a9999ec759718066f7708
+  patches:
+    - patches/0001-patch-include-paths-in-setup.py.patch
+    - patches/0002-add-missing-header.patch
+    - patches/0003-properly-point-to-triton-includes.patch
+    - patches/0004-point-to-system-CUDA-instead-of-vendored-header.patch
+
+build:
+  number: 0
+  # TODO: windows support should be available from next version;
+  #       CPU-only support still under development
+  skip: true  # [win or cuda_compiler_version == "None"]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - make
+    - cmake
+  host:
+    - python
+    - pip
+    - llvmdev =11
+    - libzlib
+  run:
+    - python
+    - filelock
+    - pytorch =*=cuda*
```

Comment:
Couldn't we just list `pytorch-gpu` here?

Reply:
Because that package is not used anymore by the `pytorch` channel, and it was only a compatibility wrapper on the side of conda-forge as well. CC @hmaarrfk

Reply:
hmmm I thought conda-forge aims to be self-contained so I don't understand why we care about other channels, though I can see this increases compatibility. Can someone from @conda-forge/staged-recipes confirm?

Reply:
We do aim to be self contained. But the the truth is we do benefit from following upstream where we can.

Leaving this syntax would benefit users building the windows versions for themselves

Reply:
Sounds good, thanks Mark. Let's leave it as is.